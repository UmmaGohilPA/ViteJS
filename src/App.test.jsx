import React from "react";
import renderer from "react-test-renderer";
import { expect, test } from "vitest";
import App from "./App";

test("About renders a snapshot properly", () => {
  const tree = renderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});
