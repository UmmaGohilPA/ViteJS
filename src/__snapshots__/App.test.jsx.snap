// Vitest Snapshot v1

exports[`About renders a snapshot properly 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="/src/logo.svg"
    />
    <p>
      Hello Vite + React!
    </p>
    <p>
      <button
        onClick={[Function]}
        type="button"
      >
        count is: 
        0
      </button>
    </p>
    <p>
      Edit 
      <code>
        App.jsx
      </code>
       and save to test HMR updates.
    </p>
    <p>
      <a
        className="App-link"
        href="https://reactjs.org"
        rel="noopener noreferrer"
        target="_blank"
      >
        Learn React
      </a>
       | 
      <a
        className="App-link"
        href="https://vitejs.dev/guide/features.html"
        rel="noopener noreferrer"
        target="_blank"
      >
        Vite Docs
      </a>
    </p>
    <p>
      Reminder: Add vitest config and sample test 
    </p>
  </header>
</div>
`;
